### DISCUSSION TITLE
Impressive

### DISCUSSION THREAD

**Discussion by barotto** [dbalsom/marty#37] [General]
Hi, I just want to say that MartyPC is an exceptional software that sets a new standard for IBM PC emulation.
I know from personal experience how much work there is in building a PC emulator. The fact that you achieved these results in such a short period of time is even more impressive.
Looking forward to reading your blog for tips and inspiration on how to effectively improve my work.
Cheers.

**Comment by dbalsom** [comment]
Thank you for the kind words. I think IBMulator is one of the more impressive emulators as well. I think we are both trying to emulate a particular machine as accurately as possible out of honest affection for the hardware. 

I love the interface you have showing the whole machine chassis and bezel and the system sounds. 

If you ever wanted to expand to the 5160/8088 machines, I plan to put up some more resources about the prefetch/BIU algorithm as well as publish a JSON test suite for the 8088.
